pysparkle/pysparkle/etl/etl.py

```diff
@@ -0,0 +1,66 @@
+# Bronze to Silver transformations.
+import logging
+from pathlib import Path
+from typing import Any, Optional
+
+from pyspark.sql import SparkSession
+
+from pysparkle.config import DEFAULT_BRONZE_CONTAINER, DEFAULT_SILVER_CONTAINER
+from pysparkle.pyspark_utils import get_spark_session, read_datasource, save_dataframe_as_delta
+from pysparkle.storage_utils import check_env, get_adls_file_url, set_spark_properties
+
+logger = logging.getLogger(__name__)
+
+
+def save_files_as_delta_tables(
+    spark: SparkSession,
+    input_files: list[str],
+    datasource_type: str,
+    spark_read_options: Optional[dict[str, Any]] = None,
+    bronze_container: str = DEFAULT_BRONZE_CONTAINER,
+    silver_container: str = DEFAULT_SILVER_CONTAINER,
```

Comment:
As above - I think we could just have `source_container` and `destination_container`. Also if we do this, I wouldn't leave the defaults in there, and require they get passed each time.